### PR TITLE
TOOL-352 Add 6.6.6, 7.1.4 and 7.1.5

### PR DESCRIPTION
**Pull request by mcpaddy**

### CODE REVIEW

File: Top_Level_Vagrantfile [hunk 1]
```diff
@@ -305,9 +308,9 @@ couchbase_download_links = {
   "ubuntu20"   => "#{couchbase_releases}/#{version}/couchbase-server-#{edition}_#{version}-ubuntu20.04_amd64",
 }
 
-default_number_of_nodes = 4
+default_number_of_nodes = 5
```
Comment (malarky) [inline Top_Level_Vagrantfile:311]: Did you mean to include these?

File: Top_Level_Vagrantfile [hunk 2]
```diff
@@ -305,9 +308,9 @@ couchbase_download_links = {
...
 default_RAM_in_MB = 1024
-default_number_of_cpus = 1
+default_number_of_cpus = 2
```
Comment (malarky) [inline Top_Level_Vagrantfile:313]: And this?

File: Top_Level_Vagrantfile (hunk 1 above)
Comment (mcpaddy) [reply Top_Level_Vagrantfile:311]: Not in this patch I was not :P

**Review by malarky — APPROVED** [review]
:shipit:

**Pull request merged**